### ISSUE TITLE
Admissions - Extend local event content type feature split

### ISSUE THREAD

**Issue by joewhitsitt** [uiowa/uiowa#3173]
Split from https://github.com/uiowa/uiowa/issues/2396
Follow-up to #3170

# Story

As a site owner of Admissions, I can use extended features off of the local event content type feature split

# Proposed solution

### Content Type

- [x] Virtual Event designation? Could the logic just look for Virtual Event link?
- [x] Virtual Featured event? Could this just be "Sticky"?
- [ ] ~~Counselors - Reference an admissions person with type "counselor".~~ https://github.com/uiowa/uiowa/issues/2487
- [x] Full Calendar based view page tab display split from views.view.events - filter on event category

**Comment by joewhitsitt** [comment]
Did we need to talk events with Admissions again before starting this? Seemed like the definition of event changed a bit when we told them what we were building.

**Comment by bspeare** [comment]
In last week's meeting, Scott mentioned that he wanted to talk about it this week.

**Comment by joewhitsitt** [comment]
I don't think the discussion at the Admissions meeting blocks or changes this work.

**Comment by joewhitsitt** [comment]
Removed https://github.com/uiowa/uiowa/issues/2727 from this since it is a separate task and next up in the priority list.
